### PR TITLE
updated documentation

### PR DESCRIPTION
**Pull request by ryachen01**
Made some minor updates to the main documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,14 +188,21 @@ the repository is located at `/opt/thingengine` and the local state directory is
 
 ### Step 5: the web frontend
 
+First set `SECRET_KEY` in your environment:
+
+```sh
+SECRET_KEY=secret
+```
+`SECRET_KEY` is used to hash the session. Choose a secure secret to prevent session hijaking.
```
Comment (gcampax) [inline README.md]: Nice, I had forgotten about this.
By the way, there are a couple more secret keys that are needed:
- JWT_SECRET_KEY: used for OAuth access tokens, same format as SECRET_KEY
- AES_SECRET_KEY: used for 2-factor keys, must be exactly 32 hex characters

Comment (ryachen01) [reply README.md]: Where exactly should these keys be implemented?

**Comment by ryachen01** [comment]
Where should those secret keys be implemented?

File: README.md (hunk 1 above)
Comment (gcampax) [reply README.md]: Environment variables, same as `SECRET_KEY`

**Comment by gcampax** [comment]
Thank you, merged!

**Pull request merged**